svelte/dashboard.svelte

```diff
@@ -238,21 +306,32 @@
               <div>No positions</div>
             {/if}
           </div>
+          <div class="fs-bottom-container">
+            <div id="healthFactorInfoORG" class="healthfactorinfo">
+              <div class="hfcontents origin" style="display: block;">
+                <p class="textlightmode rates fs-hf">
+                  Current Health Factor : {_hf(healthFactorAll, 18)}
+                </p>
+                <p class="textlightmode rates fs-hf">
+                  <span style="margin-right: 21px;"> Next Health Factor: </span>
+                  {_hf(healthFactorUnchecked, 18)}
+                </p>
+              </div>
+            </div>
+            <div id="clearALL" class="secondarybutton cursor-pointer">
+              <div
+                on:click={handleReGet}
+                id="refreshFlashPos"
+                class="textlightmode button"
+              >
+                Refresh Dashboard
+              </div>
+            </div>
+          </div>
```

Comment:
Moved inside the `if` so that it doesn't show up before dashboard is loaded

Reply:
no it's on purpose, health factor is calculated before destination is defined